requirements.txt

```diff
@@ -1,7 +1,11 @@
 requests==2.18.4
 six==1.11.0
 tenacity==4.10.0
+google-cloud-storage==1.12.0
 oauth2client==4.1.2
 google-auth==1.5.1
 google-cloud==0.34.0
 python-dateutil==2.7.3
+pandas==0.23.4
+matplotlib==2.2.3
+numpy==1.15.2
```

Comment:
Ooo we shouldn't add these to the `requirements.txt` for the main package. Can you make a new one that lives next to your scripts to contain the specific things they need?

Reply:
I'd also move the lines you added in the last PR to the new requirements file, my bad for not catching it then.
```
google-auth==1.5.1
google-cloud==0.34.0
python-dateutil==2.7.3
```